controllers/flowlogspipeline/flp_common_objects.go

```diff
@@ -773,6 +774,57 @@ func (b *builder) clusterRoleBinding(ck ConfKind, mono bool) *rbacv1.ClusterRole
 	}
 }
 
+func (b *builder) buildLokiClusterRoles() []rbacv1.ClusterRole {
+	if b.desired.Loki.Mode == flowslatest.LokiModeLokiStack {
+		return []rbacv1.ClusterRole{
+			{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: constants.LokiCRWriter,
+				},
+				Rules: []rbacv1.PolicyRule{{
+					APIGroups:     []string{"loki.grafana.com"},
+					Resources:     []string{"network"},
+					ResourceNames: []string{"logs"},
+					Verbs:         []string{"create"},
+				}},
+			},
+			{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: constants.LokiCRReader,
+				},
+				Rules: []rbacv1.PolicyRule{{
+					APIGroups:     []string{"loki.grafana.com"},
+					Resources:     []string{"network"},
+					ResourceNames: []string{"logs"},
+					Verbs:         []string{"get"},
+				}},
+			},
```

Comment:
Should we put that guy on console plugin side when enabled ?
Also the role binding only mention the writer, not the reader :thinking:

Reply:
(done creating a `loki` package via [4d5bc3e](https://github.com/netobserv/network-observability-operator/pull/474/commits/4d5bc3eefb374ea173cc7872dc7057b6b71d578d))

Reply:
thanks !